### PR TITLE
Added CODEOWNERS and MAINTAINERS files

### PR DESCRIPTION
**Pull request by toddysm**
Added CODEOWNERS and MAINTAINERS files for the new repository (ref: https://github.com/notaryproject/.github/issues/30) 

Once the PR is approved we should create a GH Team for the repo maintainers.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Repo-Level Owners (in alphabetical order) including Org-level maintainers
+* @duffney @FeynmanZhou @gokarnm @iamsamirzon @justincormack @niazfk @priteshbandi @shizhMSFT @stevelasker @toddysm @vaninrao10 @yizha1
```
Comment (Two-Hearts) [inline CODEOWNERS:2]:
```suggestion
* @duffney @FeynmanZhou @gokarnm @iamsamirzon @justincormack @niazfk @priteshbandi @shizhMSFT @stevelasker @toddysm @Two-Hearts @vaninrao10 @yizha1
```

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
+Yi Zha <yizha1@microsoft.com> (@yizha1)
+
+# Repo-Level Maintainers (in alphabetical order)
+# Note: This is for the notaryproject/notation repo
+Josh Duffney <jduffney@microsoft.com> (@duffney)
+Shiwei Zhang <shizh@microsoft.com> (@shizhMSFT)
```
Comment (Two-Hearts) [inline MAINTAINERS:16]:
```suggestion
Josh Duffney <jduffney@microsoft.com> (@duffney)
Patrick Zheng <patrickzheng@microsoft.com> (@Two-Hearts )
Shiwei Zhang <shizh@microsoft.com> (@shizhMSFT)
```

**Review by Two-Hearts — COMMENTED** [review]
@toddysm As I'm working on notation Github Actions. Please add me into the lists.

**Review by Two-Hearts — APPROVED** [review]
LGTM

File: MAINTAINERS [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+Milind Gokarn <gokarnm@amazon.com> (@gokarnm)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
+Yi Zha <yizha1@microsoft.com> (@yizha1)
+
+# Repo-Level Maintainers (in alphabetical order)
+# Note: This is for the notaryproject/notation repo
```
Comment (yizha1) [inline MAINTAINERS:14]:
```suggestion
# Note: This is for the notaryproject/notation-action repo
```

**Review by yizha1 — CHANGES_REQUESTED** [review]
Thanks @toddysm.

File: MAINTAINERS [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Feynman Zhou <feynmanzhou@microsoft.com> (@FeynmanZhou)
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Milind Gokarn <gokarnm@amazon.com> (@gokarnm)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
+Yi Zha <yizha1@microsoft.com> (@yizha1)
```
Comment (shizhMSFT) [inline MAINTAINERS:11]: General questions:
1. Should org-level maintainers be in every repository?
2. Should org-level maintainers be CODEOWNERS in every repository? It is worth noting that being a code owner means that you will be notified and requested for code review.
3. Are org-level maintainers counted for repo-level voting? e.g. Publish a new release.
4. What if an org-level maintainer is inactive for a long time for a certain repository but he/she is indeed active in other repositories?
5. Is this pattern scalable for org-level maintainers?

File: MAINTAINERS [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Org-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Feynman Zhou <feynmanzhou@microsoft.com> (@FeynmanZhou)
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Milind Gokarn <gokarnm@amazon.com> (@gokarnm)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
```
Comment (priteshbandi) [inline MAINTAINERS:10]: Can we replace Vani with Samir as she is no longer working on this project.

File: MAINTAINERS (hunk 4 above)
Comment (SteveLasker) [reply MAINTAINERS:11]: The initial intent was the Org maintainers are a subset of folks to facilitate overall project support. They may not be as active in every project. 
When using GitHub teams, the permissions are configured through overlay of multiple teams, across multiple repositories. However, there's no traceability for those changes. 
So, yes, it's assumed org owners are there to collaboratively help the projects when needed. 
The reason to list org owners in each repo is redundant, as org owners are captured here: https://github.com/notaryproject/.github/blob/main/MAINTAINERS
So, IMO, org-owners are not required in each repo.

File: MAINTAINERS (hunk 5 above)
Comment (yizha1) [reply MAINTAINERS:10]: @priteshbandi the following document is our current guidance on `Maintainers`. Please take a look, and we can follow the process to nominate Samir, and move Vani to emeritus. 

https://github.com/notaryproject/.github/blob/main/GOVERNANCE.md#notary-org-maintainers

**Comment by toddysm** [comment]
I have updated the files as per the discussions we had in the community meetings.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

File: CODEOWNERS [hunk 6]
```diff
@@ -0,0 +1,3 @@
+# Repo-Level Owners (in alphabetical order). Org-level maintainers are included in the linked team at the end
+# For Org level maintainers, refer to https://github.com/notaryproject/.github/blob/main/CODEOWNERS
+* @duffney @shizhMSFT @Two-Hearts @notaryproject-org-maintainers
```
Comment (shizhMSFT) [inline CODEOWNERS:3]: It seems that @notaryproject-org-maintainers isn't correct. Should it be @notaryproject/notaryproject-org-maintainers?

**Review by shizhMSFT — CHANGES_REQUESTED** [review]

**Review by priteshbandi — COMMENTED** [review]
LGTM

File: MAINTAINERS (hunk 5 above)
Comment (toddysm) [reply MAINTAINERS]: @priteshbandi we need to have official issue filed to put @vaninrao10 in emeritus and add @iamsamirzon. If you want to submit the proposal in the `.github` repository we can start the process

File: MAINTAINERS (hunk 4 above)
Comment (toddysm) [reply MAINTAINERS]: Agreed, this should be fixed in the latest commit.

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**